README.md

```diff
@@ -1,29 +1,37 @@
 # pah-fm
 
 
-# Build project
+## Run project (in background)
 
 ```
-make build
+make start
 ```
 
-# Run project
-
+## Stop project
 ```
-docker-compose up
+make stop
 ```
 
-# Run project locally for testing
+## Attach to logs
+```
+make logs
 ```
-docker-compose up -d db
 
-virtualenv env -p /usr/local/bin/python3
-source env/bin/activate
+## Run linters
+```
+make lint
+```
 
-pip install -r backend/requirements/dev.txt
+## Run tests
+```
+make test
+```
 
-make manage CMD=createsuperuser
+## Debug backend (Django)
+```
+make debug-backend
 ```
+After finishing debugging detach from shell using *CTRL*+`p` and *CTRL*+`q`.
```

Comment:
Can we please also add section on accessing frontend and admin pages?

Reply:
As I wrote in description I have a plan to do nginx cleanup and prefer not to mix these two things.

BTW what does it mean?
> accessing frontend
 Do You want to attach to node process, that run webpack builder? What for?

Reply:
I don't know what this means. Where is this from?

Reply:
You just wrote about "accessing frontend and admin pages" - I assumed, that You want to access proccess that run in frontend docker container responsible for hot reload of Vue changes.

If You were refering just what to write in browser address bar, I would like to handle it with nginx cleanup (which I'm working on).